### DISCUSSION TITLE
Vortex induced vibration of cylinder

### DISCUSSION THREAD

**Discussion by wWL2771140158** [xcompact3d/Incompact3d#126] [Q&A] [answered]
Hello, I'm going to use the Cylinder module to study the vortex induced vibration of the cylinder. How can I add the mass and damping ratio of the cylinder and other parameters to the dynamic equation of the cylinder, or is there any module related to vortex induced vibration that can be directly called? thank you!

**Comment by slaizet** [comment, marked as answer]
Hi,
I am afraid the VIV features are not yet available in the master branch so you will have to implement them yourself.
Best,
Sylvain

**Reply by wWL2771140158** [reply]
OK, thank you for your suggestion！
